### PR TITLE
Updated the Claim Hat e-mail to match the latest designs

### PR DESCRIPTION
**Pull request by Ddeak**

### CODE REVIEW

**Review by whiteshadow-gr — APPROVED** [review]

File: hat/app/org/hatdex/hat/api/controllers/Authentication.scala [hunk 1]
```diff
@@ -494,7 +494,7 @@ class Authentication @Inject() (
                         )
                         Done
                     }
-                  mailer.passwordChanged(token.email)
+                  mailer.hatClaimed(token.email, user.name)
```
Comment (whiteshadow-gr) [inline hat/app/org/hatdex/hat/api/controllers/Authentication.scala:497]: @augustinas a bit of help here, `user.name` is it going to be `something.dataswift.dev`? It's a `HatUser` type

Comment (whiteshadow-gr) [reply hat/app/org/hatdex/hat/api/controllers/Authentication.scala:497]: It's actually supposed to be `https://hat.dataswift.dev` to lead them to log in page. Or the sign in page in general. So this has to change definitely I am not sure how to get the hat address. Or actually redirect them to PDA Auth to log in? 🤷‍♂️

Comment (augustinas) [reply hat/app/org/hatdex/hat/api/controllers/Authentication.scala:497]: Link can have redirect to the authentication service or direct to PDA's password screen. The question I would ask is what application ID it should carry in the redirect? What is the intended action after the button click?

File: hat/app/org/hatdex/hat/utils/Mailer.scala [hunk 2]
```diff
@@ -112,6 +112,14 @@ trait HatMailer extends Mailer {
     )(implicit m: MessagesApi,
       lang: Lang,
       server: HatServer): Done
+
+  // no changes
+  def hatClaimed(
```
Comment (augustinas) [inline hat/app/org/hatdex/hat/utils/Mailer.scala:117]: The code can be made more readable by semantically aligning it with `verifyEmail` method. E.g. `emailVerified` or similar.

File: hat/app/org/hatdex/hat/utils/Mailer.scala [hunk 3]
```diff
@@ -206,4 +214,21 @@ class HatMailerImpl @Inject() (
...
+  // data is ok
+  def hatClaimed(email: String,
+                 loginLink: String
+    )(implicit messages: MessagesApi,
+                   lang: Lang,
+                   server: HatServer): Done = {
+    sendEmail(email)(
+      from = emailFrom,
+      subject = messages("email.dataswift.auth.subject.verifyEmail"),
+      bodyHtml = views.html.mails.emailHatClaim(email, loginLink),
```
Comment (augustinas) [inline hat/app/org/hatdex/hat/utils/Mailer.scala:229]: The code can be made more readable by changing template name to align with current vocabulary. Term "claim" is being faded out.

**Review by augustinas — COMMENTED** [review]
Minor comments regarding readability.

**Pull request merged**